pkg/apis/applicationnetworking/v1alpha1/accesslogpolicy_types.go

```diff
@@ -95,8 +95,8 @@ func (p *AccessLogPolicy) GetNamespacedName() types.NamespacedName {
 
 func (pl *AccessLogPolicyList) GetItems() []core.Policy {
 	items := make([]core.Policy, len(pl.Items))
-	for i, item := range pl.Items {
-		items[i] = &item
+	for i := range pl.Items {
+		items[i] = &pl.Items[i]
 	}
```

Comment:
Previous code `items[i] = &item` has a bug that mess up the item pointers. it must use `items[i] = &pl.Items[i]`

Reply:
Thanks for catching this. What was the bug doing?